jsonalchemy/validator.py

```diff
@@ -113,9 +115,23 @@ def _validate(self, document, schema=None, update=False):
                 if not self.allow_unknown:
                     self._error(field, errors.ERROR_UNKNOWN_FIELD)
 
+        # Remove special rules, as they can cause error in Cerberus validation.
```

Comment:
Cerberus gets from us whole schema, with special rules inside. That's why it fails.

After #13 was merged, this is obsolete anyways.